ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-ref.yaml

```diff
@@ -13,6 +13,10 @@ ref:
   - namespace: test-credentials
     name: ci-ibmcloud
     mount_path: /var/run/vault/ibmcloud
+  env:
```

Comment:
How about keeping the same concept with other cloud providers like AWS to use `SIZE_VARIANT = compact` to indicate it's a compact 3-node cluster. 

https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh#L143-L145

if [[ "${SIZE_VARIANT}" == "compact" ]], then we can set replicas of worker =0 and other master/worker parameters to `MACHINE_POOL_OVERRIDES` in a separate judgement statement.

Reply:
good idea, I will work on this later.

Reply:
@gpei test pass, please help review on the code.